frontend/src/app/forms/forms.page.html

```diff
@@ -0,0 +1,55 @@
+<ion-header class="ion-no-border">
+  <ion-toolbar>
+    <ion-grid>
+      <ion-row>
+        <ion-col class="font30">
+          <ion-icon name="close" (click)="onClose()"></ion-icon>
+        </ion-col>
+        <ion-col class="ion-text-center ion-float-center progresstext">
+          Step {{currentMenu.id}}/ {{totalSteps}}
+        </ion-col>
+        <ion-col>
+          <ion-menu-button
+            class="ion-float-right font30"
+            autoHide="true"
+            menuClose
+            id="main"
+          >
+            <ion-icon name="list"></ion-icon>
+          </ion-menu-button>
+        </ion-col>
+      </ion-row>
+    </ion-grid>
+
+    <ion-progress-bar value="{{progressPercentage}}"></ion-progress-bar>
+  </ion-toolbar>
+</ion-header>
+
+<app-side-menu
+  [sideMenuList]="sideMenuList"
+  (pageProgressStatusCallBack)="onFormStepsNavigation($event)"
+>
+</app-side-menu>
+
+<ion-content class="ion-padding">
+  <ng-container [ngSwitch]="currentMenu.selector">
+    <form-basic-info
+      (pageProgressStatusCallBack)="pageProgressStatusCallBack($event)"
```

Comment:
You might want to consider renaming 
```
(pageProgressStatusCallBack)="pageProgressStatusCallBack($event)"
```
to something like
```
(pageProgressStatusChange)="pageProgressStatusCallBack($event)"
```
or similar. Simply to sound more like an event. But that's up to you.

Reply:
We will discuss about naming conventions and change it in future if necessary.